### PR TITLE
fix #79047 upload-4ever.com

### LINKED ISSUE #79047: www.upload-4ever.com
### Issue URL (Incorrect Blocking)
 [https://www.upload-4ever.com/7gabm9026ivh](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.upload-4ever.com%2F7gabm9026ivh)
### Comment
Username: @chigarow
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Mac v2.5.3.955 release
Browser: | Firefox
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>EasyList, <br/>Peter Lowe's Blocklist<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy, <br/>Fanboy's Enhanced Tracking List<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/> AdGuard DNS<br/><br/><b>Language-specific:</b><br/> AdBlockID
Custom filters: | https://block.energized.pro/spark/formats/filter
Userscripts: | https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.meta.js,<br>https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.meta.js
Referrer: | [https://www.upload-4ever.com/7gabm9026ivh](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.upload-4ever.com%2F7gabm9026ivh)

### PR DESCRIPTION
**Pull request by wout-codes**
#79047

### CODE REVIEW

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/79047
+upload-4ever.com#?#div[class^="my-"]:has(> ins.adsbygoogle)
```
Comment (zloyden) [inline EnglishFilter/sections/css_extended.txt:5]: I'd use `:upward` `upload-4ever.com#?#div[class^="my-"] > ins.adsbygoogle:upward(1)`

Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt:5]: Same thing.

Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt:5]: We are all right in our own perspective.

Comment (Yuki2718) [reply EnglishFilter/sections/css_extended.txt:5]: Not the same but in this case :has writing is fine as the number of nodes to evaluate is 2 vs. 3 against :upward writing.

Comment (zloyden) [reply EnglishFilter/sections/css_extended.txt]: I know this, but after some cases, I prefer to use `:has` only when necessary.

Comment (Yuki2718) [reply EnglishFilter/sections/css_extended.txt]: @wout-codes If you wonder, I talked about performance than what are hidden.

**Pull request merged**